client/src/Components/RecipeTab/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { Recipe } from 'lib/types'
+
+interface Props {
+  recipe: Recipe
+}
+
+const RecipeTab: React.FC<Props> = ({ recipe }: Props) => {
+  const bgcolor = 'bg-' + recipe.bgcolor
+  const headcolor = 'text-' + recipe.headcolor
+  const textcolor = 'text-' + recipe.textcolor
+  const buttoncolor = 'bg-' + recipe.headcolor
+  const buttontextcolor = 'text-' + recipe.buttontextcolor
+
+  return (
+    <div className={'grid gap-4 grid-cols-1 sm:grid-cols-2 m-10 p-10 rounded-xl ' + bgcolor}>
+      <div>
+        <h1 className={'text-4xl font-semibold font-serif ' + headcolor}>
+          {recipe.name}
+        </h1>
+        <h3 className={'text-2xl font-serif ' + headcolor}>Category</h3>
```

Comment:
Is it ok in my most recent changes that I've got those h3 headings smaller in actual size than the h2 ones? I thought it would be better to emphasize the important sections (equipment and ingredients). I can easily change the tags though no worries

Reply:
if you want the modify the text size, use https://tailwindcss.com/docs/font-size#class-reference